ci-operator/step-registry/redhat-chaos/kubevirt-outage/redhat-chaos-kubevirt-outage-commands.sh

```
@@ -73,5 +73,10 @@ rc=$?
 if [[ $TELEMETRY_EVENTS_BACKUP == "True" ]]; then
     cp /tmp/events.json ${ARTIFACT_DIR}/events.json
 fi
+
+
+if [[ -f /home/krkn/kraken/kraken.report.pdf ]]; then
+  cp /home/krkn/kraken/kraken.report.pdf ${ARTIFACT_DIR}/kraken.report.pdf
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)